src/transformers/models/layoutlmv2/feature_extraction_layoutlmv2.py

```diff
@@ -102,6 +102,9 @@ class LayoutLMv2FeatureExtractor(FeatureExtractionMixin, ImageFeatureExtractionM
             Only has an effect if :obj:`do_resize` is set to :obj:`True`.
         apply_ocr (:obj:`bool`, `optional`, defaults to :obj:`True`):
             Whether to apply the Tesseract OCR engine to get words + normalized bounding boxes.
+        ocr_lang (:obj:`Optional[str]`, `optional`, defaults to :obj:`None`):
```

Comment:
```suggestion
        ocr_lang (:obj:`Optional[str]`, `optional`):
```
Sorry for commenting again, but "defaults to None" is omitted in the library.